.github/workflows/publish-meta-images.yaml

```diff
@@ -0,0 +1,249 @@
+          VERSION="${{ github.event.client_payload.version }}"
+          echo "STAGE=unstable" >> $GITHUB_ENV
+          echo "FULL_TAG=$(date -u '+%Y%m%d.%H%M')" >> $GITHUB_ENV
+          echo "SHORT_TAG=$(date -u '+%Y%m%d')" >> $GITHUB_ENV
+
+          if [[ "${TYPE,,}" =~ 'web' ]]; then
+            WEB_VERSION="${VERSION}"
+            SERVER_VERSION="${VERSION%.*}"
+          else
+            WEB_VERSION="${VERSION%.*}"
+            SERVER_VERSION="${VERSION}"
+          fi
+
+          echo "SERVER_VERSION=$(echo ${SERVER_VERSION} | grep -Eo '[0-9]+\.[0-9]+(\.[0-9])?')" >> $GITHUB_ENV
+          echo "WEB_VERSION=$(echo ${WEB_VERSION} | grep -Eo '[0-9]+\.[0-9]+(\.[0-9])?')" >> $GITHUB_ENV
+
+      - name: Setup Manual Environment
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |-
+          SERVER_VERSION="${{ github.event.inputs.SERVER_TAG }}"
+          WEB_VERSION="${{ github.event.inputs.WEB_TAG }}"
+          IS_STABLE="${{ github.event.inputs.IS_STABLE }}"
+
+          if [[ "${IS_STABLE,,}" =~ 'true' ]]; then
+            echo "STAGE=stable" >> $GITHUB_ENV
+            echo "FULL_TAG=$(echo ${SERVER_VERSION} | grep -Eo '[0-9]+\.[0-9]+\.[0-9]')" >> $GITHUB_ENV
+            echo "SHORT_TAG=$(echo ${SERVER_VERSION} | grep -Eo '[0-9]+\.[0-9]+')" >> $GITHUB_ENV
+          else
+            echo "STAGE=unstable" >> $GITHUB_ENV
+            echo "FULL_TAG=$(date -u '+%Y%m%d.%H%M')" >> $GITHUB_ENV
+            echo "SHORT_TAG=$(date -u '+%Y%m%d')" >> $GITHUB_ENV
+          fi
+
+          echo "SERVER_VERSION=$(echo ${SERVER_VERSION} | grep -Eo '[0-9]+\.[0-9]+\.[0-9]')" >> $GITHUB_ENV
+          echo "WEB_VERSION=$(echo ${WEB_VERSION} | grep -Eo '[0-9]+\.[0-9]+\.[0-9]')" >> $GITHUB_ENV
+
+
+      - name: Setup sariables
+        id: variables
+        run: |-
+          echo "::set-output name=stage::${STAGE}"
+          echo "::set-output name=server-version::${SERVER_VERSION}"
+          echo "::set-output name=web-version::${WEB_VERSION}"
+          echo "::set-output name=full-tag::${FULL_TAG}"
+          echo "::set-output name=short-tag::${SHORT_TAG}"
+
+  container-images:
+    name: Build Meta Container Images
+    runs-on: ubuntu-latest
+    needs:
+      - setup-vars
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - image-tag: amd64
+            image-config: amd64
+          - image-tag: arm64
+            image-config: arm64
+          - image-tag: armhf
+            image-config: arm32
+
+    env:
+      STAGE: ${{ needs.setup-vars.outputs.stage }}
+      SERVER_VERSION: ${{ needs.setup-vars.outputs.server-version }}
+      WEB_VERSION: ${{ needs.setup-vars.outputs.web-version }}
+      FULL_TAG: ${{ needs.setup-vars.outputs.full-tag }}
+      SHORT_TAG: ${{ needs.setup-vars.outputs.short-tag }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install qemu dependency
```

Comment:
Why not use the same action we use in Vue? It should simplify this setup a lot

Reference: https://github.com/jellyfin/jellyfin-vue/blob/master/.github/workflows/docker-release.yml#L30